spec/lib/findingaids/ead/component_spec.rb

```diff
@@ -175,13 +175,17 @@
       end
     end
 
+    describe "#bread_crumb" do
```

Comment:
@eshadatta Call this `breadcrumb` instead. If there is breadcrumb and bread_crumb is creates confusion